src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
According to the checklist, you should use a loop for creating several objects of the same class. Consider using a loop to create the three `Ball` objects instead of creating each one individually. This will make your code more concise and easier to maintain.